lighthouse-core/lib/lh-error.js

```diff
@@ -172,14 +172,6 @@ const ERRORS = {
     message: UIStrings.pageLoadFailedWithStatusCode,
     lhrRuntimeError: true,
   },
-  /* Used when security error prevents page load.
```

Comment:
re: discussion in #8865, I don't think we should get rid of this LighthouseError. I think it should be repurposed as the `lhr.runtimeError` in case of interstitial detected instead of it being `FAILED_DOCUMENT_REQUEST` bubbling up from one of the audits

Reply:
reverted, followup can bring interstitial detection magic back to make this INSECURE_DOCUMENT_REQUEST again :)